### ISSUE TITLE
doesnt work on 5.2

### ISSUE THREAD

**Issue by xcloudplatform** [adamrehn/ue4cli#55]
```
% ls /Users/Shared/Epic\ Games/UE_5.2/Engine/Binaries/Mac/BuildPatchTool --version
ls (GNU coreutils) 9.1
Copyright (C) 2022 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by Richard M. Stallman and David MacKenzie.

﷽      ~/unreal-rust/example/RustExampleمَّنْ عَمِلَ صَٰلِحا ࣰ فَلِنَفْسِهِۦۖ وَمَنَ اَسَآءَ فَعَلَيْهَاۖ وَمَا رَبُّكَ بِظَلَّٰم ࣲ لِّلْعَبِيدِۖ 
% ue4 build Development Editor                      239234c [239234c] untracked 6213 12:53:12

Traceback (most recent call last):
  File "/opt/homebrew/bin/ue4", line 8, in <module>
    sys.exit(main())
  File "/opt/homebrew/lib/python3.10/site-packages/ue4cli/cli.py", line 222, in main
    SUPPORTED_COMMANDS[command]['action'](manager, args)
  File "/opt/homebrew/lib/python3.10/site-packages/ue4cli/cli.py", line 76, in <lambda>
    'action': lambda m, args: m.buildDescriptor(os.getcwd(), args.pop(0) if (len(args) > 0 and args[0].startswith('-') == False) else 'Development', args.pop(0) if (len(args) > 0 and args[0].startswith('-') == False) else 'Editor', args),
  File "/opt/homebrew/lib/python3.10/site-packages/ue4cli/UnrealManagerBase.py", line 351, in buildDescriptor
    if noTools == False and self.isInstalledBuild() == False and self.isProject(descriptor) and target == 'Editor':
  File "/opt/homebrew/lib/python3.10/site-packages/ue4cli/UnrealManagerBase.py", line 98, in isInstalledBuild
    sentinelFile = os.path.join(self.getEngineRoot(), 'Engine', 'Build', 'InstalledBuild.txt')
  File "/opt/homebrew/lib/python3.10/site-packages/ue4cli/UnrealManagerBase.py", line 60, in getEngineRoot
    self._engineRoot = self._getEngineRoot()
  File "/opt/homebrew/lib/python3.10/site-packages/ue4cli/UnrealManagerBase.py", line 620, in _getEngineRoot
    return self._detectEngineRoot()
  File "/opt/homebrew/lib/python3.10/site-packages/ue4cli/UnrealManagerDarwin.py", line 25, in _detectEngineRoot
    if version < parse_version('5.0.0'):
NameError: name 'version' is not defined
```

**Comment by sleeptightAnsiC** [comment]
same as https://github.com/adamrehn/ue4cli/pull/50 and https://github.com/adamrehn/ue4cli/issues/57
small change in UnrealManagerDarwin.py fixes it, e.g. https://github.com/sleeptightAnsiC/ue4cli/commit/cc4773751032623f38cdda12e252a593db909efd

**Comment by adamrehn** [comment]
Fixed by the merge of #58.
